redfish-core/lib/license_service.hpp

```diff
@@ -99,38 +99,34 @@ inline void
 
     if (status == "com.ibm.License.LicenseManager.Status.ActivationFailed")
     {
-        // TODO: Need to return appropriate redfish error
         BMCWEB_LOG_ERROR << "LicenseActivationStatus: ActivationFailed";
-        messages::internalError(asyncResp->res);
+        messages::installFailed(asyncResp->res);
```

Comment:
this takes a string argument - "The reason for the license installation failure."

Reply:
done